simapp/doc.go

```diff
@@ -0,0 +1,94 @@
+/*
+# Simulation App
```

Comment:
This docs is great, though it should adhere more strictly to go doc formatting. Please see https://medium.com/@elliotchance/godoc-tips-tricks-cda6571549b for more info

Reply:
I don't follow? How does this not adhere? This is a valid proper godoc. The only difference is the usage of `//` vs `/* */`.

Reply:
Can you point out what needs to change?

Reply:
![1](https://user-images.githubusercontent.com/229356/58971228-e35d9400-87b2-11e9-997e-4fe66d85ba96.png)
![2](https://user-images.githubusercontent.com/229356/58971233-e6f11b00-87b2-11e9-8fe9-3fb986205f49.png)

Reply:
I see what you mean -- I'll amend.